src/fr/composants/avis/code.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Avis
+layout: 'layouts/component-documentation.njk'
+translationKey: 'noticeCode'
+tags: ['noticeFR', 'code']
+date: "git Last Modified"
+---
+
+## Créer un avis
+
+Utilisez le composant d’avis pour transmettre aux gens des messages importants et propres au contexte. L’avis communique divers niveaux d’urgence et différents messages à l’aide de contenu et d’éléments visuels.
+
+## Accessibilité et lignes directrices relatives au code
+
+### Définir le type d’avis
+
+Utilisez l’attribut `type` pour classer l’avis dans les catégories « Information », « Avertissement », « Danger » ou « Succès ». Ce choix détermine le style visuel de l’avis et communique l’urgence ou l’importance du message au public.
```

Comment:
Similar comment about: « Information », « Avertissement », « Danger » ou « Succès ».

Can we flag them as code, have them in lowercase and keep them in English so they match the actual values users need to pass to the component?

Reply:
Does this work for the french text @SmartMouthWords ?

```suggestion
Utilisez l’attribut `type` pour classer l’avis dans les catégories `info` (Information), `warning` (Avertissement), `danger` (Danger) ou `success` (Succès). Ce choix détermine le style visuel de l’avis et communique l’urgence ou l’importance du message au public.
```